lib/src/test/java/com/thenewboston/api/bankapi/datasource/BankDataSourceTest.kt

```diff
@@ -4,12 +4,14 @@ import com.thenewboston.common.http.NetworkClient
 import com.thenewboston.common.http.Outcome
```

Comment:
Hey @nishp77 I've refactored the unit tests to use Kotest for assertions. You can pull down the latest master to see how you would use it.